README.md

```diff
@@ -1,56 +1,48 @@
-# grants
+# dcgrants
 
-## Project setup
-### Truffle
-`$ truffle develop`
+## Development
 
-or
+This project is a monorepo with three packages:
 
-Run Ganache
-then
-`$ truffle console --network ganache`
+- `contracts` contains the smart contracts
+- `app` is a frontend
 
-`$ truffle compile`
-and
-`$ truffle migrate`--reset
+### Dependencies
 
-or
+To ensure that everyone is using the same version of nodejs on this project, [volta](https://volta.sh) is recommended!
 
-`$ truffle migrate`--reset --compile-all
+### Set your env files
 
-### Vue
-```
-npm install
+```sh
+cp app/.env.template app/.env
+nano app/.env
 
-To install the drizzle/vue-plugin you may have to do the following
-`$ npm install --legacy-peer-deps @drizzle/vue-plugin`
-`$ npm install --legacy-peer-deps vuex`
+cp contracts/.env.template contracts/.env
+nano contracts/.env
```

Comment:
A bit of a nit, but `cp` and `nano` don't exist on Windows so I'd recommend just explaining the steps in regular words so it's better documented for cross-platform/accessible to new Windows devs